po/de.po

```diff
@@ -11961,45 +12780,45 @@ msgstr ""
 msgid "You need to specify a tag name."
 msgstr "Sie müssen den Namen des Tags angeben."
 
-#: builtin/fetch.c:1592
+#: builtin/fetch.c:1594
 msgid "Negative depth in --deepen is not supported"
 msgstr "Negative Tiefe wird von --deepen nicht unterstützt."
 
-#: builtin/fetch.c:1594
+#: builtin/fetch.c:1596
 msgid "--deepen and --depth are mutually exclusive"
 msgstr "--deepen und --depth schließen sich gegenseitig aus"
 
-#: builtin/fetch.c:1599
+#: builtin/fetch.c:1601
 msgid "--depth and --unshallow cannot be used together"
 msgstr ""
 "Die Optionen --depth und --unshallow können nicht gemeinsam verwendet werden."
 
-#: builtin/fetch.c:1601
+#: builtin/fetch.c:1603
 msgid "--unshallow on a complete repository does not make sense"
 msgstr ""
 "Die Option --unshallow kann nicht in einem Repository mit unvollständiger "
```

Comment:
Wie kürzlich auf https://www.heise.de/developer/artikel/Die-vielfaeltigen-Faehigkeiten-von-Git-Teil-1-4456122.html?seite=4 prominent angemerkt wurde, ist "unvollständiger" hier während der Übersetzung und Umformulierung falsch geworden. Sollte sicherlich "vollständig" heißen, oder? Wird das bei einer Überarbeitung dieses PRs mit aktualisiert, oder soll ich einen separaten PR einreichen?

Reply:
Schon in https://github.com/ruester/git-po-de/pull/3 erledigt :-)